test/javascripts/acceptance/category-chooser-initializer-test.js

```diff
@@ -3,8 +3,8 @@ import { acceptance } from "discourse/tests/helpers/qunit-helpers";
 import selectKit from "discourse/tests/helpers/select-kit-helper";
 import { test } from "qunit";
 
-acceptance("Category Chooser Initializer", function (needs) {
-  needs.user();
+acceptance("Category Chooser Initializer for regular users", function (needs) {
+  needs.user({ staff: false, admin: false, moderator: false });
```

Comment:
Likewise here, just `staff` is necessary.

Reply:
The change is done